src/servers/src/mysql/federated.rs

```diff
@@ -344,12 +355,19 @@ mod test {
         }
 
         let query = "select version()";
-        let expected = "\
-+-----------+
-| version() |
-+-----------+
-| 8.0.26    |
-+-----------+";
+        let mut buffer = String::new();
```

Comment:
This should do the trick:
```suggestion
        let expected = format!(
            r#"+----------------+
| version()      |
+----------------+
| {}-greptime |
+----------------+"#,
            env::var("CARGO_PKG_VERSION").unwrap_or_else(|_| "unknown".to_string())
        );
        test(query, &expected);
```

Reply:
OK